Source/BSN.Commons/Infrastructure/MessageBroker/RabbitMqEventAggregator/ExchangeOptions.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+
+namespace BSN.Commons.Infrastructure.MessageBroker.RabbitMqEventAggregator
+{
+    public class ExchangeOptions
+    {
+        public string BrokerName { get; set; } = "Resa_event_bus";
+        public string BrokerType { get; set; } = "direct";
```

Comment:
Maybe, it is better to `enum` rather than string.

Reply:
done.